### PR TITLE
Explain how to correlate GitHub runner metrics and jobs

### PR DESCRIPTION
**Pull request by cg14823**
### What does this PR do?

Adds explanation on how infrastructure and GitHub CI job correlation works.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ To enable logs, follow these steps:
 
 Immediately after toggling logs collection, workflow job logs are forwarded to Datadog Logs. Note that logs are billed separately from CI Visibility. Log retention, exclusion, and indexes are configured in Logs Settings.
 
+### Infrastructure metric correlation
+
+If you are running the GitHub runners you can correlate your jobs to the host is running. To do so make sure the GitHub runner name
+matches the hostname of the machine it is running on. CI Visibility will automatically detect the runner name and link it to the
```
Comment (AdrianLC) [inline content/en/continuous_integration/setup_pipelines/github.md:68]:
```suggestion
If you are using self-hosted GitHub runners you can correlate jobs to the host that is running them. To do so make sure the GitHub runner name
matches the hostname of the machine it is running on. CI Visibility uses this to link to
```
Not sure if self-hosted is the best word for Github but _you are running the GitHub runners_ sounds a bit too informal to me

**Review by AdrianLC — COMMENTED** [review]
I think we usually don't attach screenshots because they might become outdated without us remembering to update the docs

**Comment by cg14823** [comment]
> I think we usually don't attach screenshots because they might become outdated without us remembering to update the docs

Fair I'll remove the image and add a bit more words.

**Review by AdrianLC — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 2]
```diff
@@ -62,6 +62,13 @@ To enable logs, follow these steps:
 
 Immediately after toggling logs collection, workflow job logs are forwarded to Datadog Logs. Note that logs are billed separately from CI Visibility. Log retention, exclusion, and indexes are configured in Logs Settings.
 
+### Infrastructure metric correlation
+
+If you are using self-hosted GitHub runners you can correlate jobs to the host that is running them. To do so make sure the GitHub runner name
```
Comment (jtappa) [inline content/en/continuous_integration/setup_pipelines/github.md:67]:
```suggestion
If you are using self-hosted GitHub runners, you can correlate jobs to the host that is running them. To do this, make sure the GitHub runner name
```

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 3]
```diff
@@ -62,6 +62,13 @@ To enable logs, follow these steps:
...
+matches the hostname of the machine it is running on. CI Visibility uses this to link to
+infrastructure metrics. To see the metrics just click on a job span in the trace view and in the drawer a new tab named **Infrastructure** will
```
Comment (jtappa) [inline content/en/continuous_integration/setup_pipelines/github.md:69]:
```suggestion
infrastructure metrics. To see the metrics, click on a job span in the trace view and in the window a new tab named **Infrastructure** is
```

File: content/en/continuous_integration/setup_pipelines/github.md [hunk 4]
```diff
@@ -62,6 +62,13 @@ To enable logs, follow these steps:
...
+show which contains the host metrics.
```
Comment (jtappa) [inline content/en/continuous_integration/setup_pipelines/github.md:70]:
```suggestion
shown which contains the host metrics.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**